content/en/tracing/trace_retention_and_ingestion/usage_metrics.md

```diff
@@ -5,24 +5,40 @@ description: "Learn how to monitor your Tracing without Limits usage."
 
-Datadog provides an out-of-the-box [Usage Dashboard][3] for monitoring your APM usage, as well as your ingested and Indexed Spans.
+### Trace Analytics Usage Dashboard
 
-To create a custom dashboard or monitor, the key metrics to use are:
+{{< img src="tracing/trace_indexing_and_ingestion/usage_metrics/AnalyticsDashboardOverview.png" style="width:100%;" alt="Tracing without Limits Usage Dashboard" >}}
+
+Datadog provides an out-of-the-box [Usage Dashboard][5] for monitoring your APM usage, as well as your ingested and indexed span volumes.
+
+Each metric on this dashboard is powered by one of the below three Datadog standard metrics available in your account.
 
  - `datadog.estimated_usage.apm.ingested_bytes`
  - `datadog.estimated_usage.apm.ingested_spans`
  - `datadog.estimated_usage.apm.indexed_spans`
 
+Each of these metrics is also tagged by `env` and `service` to allow easy detection of what services and in which environments any fine-tuning of ingestion or indexing controls should be done in.  Use these metrics within our default dashboard or create your own dashboards and monitors to detect retention filter misconfiguration or set thresholds for monitors.
+
+The default [Trace Analytics Dashboard][5] groups are below, including the widgets each contains:
+
+| Group                         | Widgets                                                                                            |
+| ------------------------------ | ------------------------------------------------------------------------------------------------------ |
+| Summary | Shows the APM Host count currently and as a timeseries, along with ingested and indexed spans for an at-a-glance determination if filters need to be tuned.  |
+| Ingestion overview           | Shows a detailed view of ingestion, including bytes per second, total bytes ingested during the time period, and a breakdown of ingestion by `env`.  The same graphs are again available for spans instead of bytes.  **Note:** Billing is based on bytes.|
+| Indexing overview          | Shows a detailed view of indexed spans, including a timeseries of spans indexed per second, total spans indexed over the period, and a breakdown of indexed spans by `env`.  |
+| Per env breakdown |   For each environment, a top list is provided of the ingested and indexed spans from that environment, as well as a top list of what this ratio is per environment.                                                         |
+| Per service breakdown         | For each service, a top list is provided of the ingested and indexed spans from that service, as well as a top list of what this ratio is per service.       |
+| Per env and service combination breakdown         | For each unique combination of `env` and `service`, a top list is provided of the ingested and indexed spans from that environment and service, as well as a top list of what this ratio is per combination. |
+
```

Comment:
This seems like we're documenting the UI, which should be self-explanatory, and this table is destined to be wrong almost before it's published. For example, "Summary" is not the heading on that section of the dashboard. I suggest we remove the table and only describe the dashboard generally, not particularly. Can say things like "Billing is based on bytes"

Reply:
Updated - what do you think?